aws_xray_sdk/ext/bottle/plugin.py

```diff
@@ -0,0 +1,105 @@
+from bottle import request, response, SimpleTemplate
+
+from aws_xray_sdk.core.lambda_launcher import check_in_lambda, LambdaContext
+from aws_xray_sdk.core.models import http
+from aws_xray_sdk.core.utils import stacktrace
+from aws_xray_sdk.ext.util import calculate_sampling_decision, \
+    calculate_segment_name, construct_xray_header, prepare_response_header
+
+
+class XRayPlugin(object):

```

Comment:
Can we name this class as XRayMiddleware and also add this comment: 
```
"""
    Middleware that wraps each incoming request to a segment.
""" 
```
? just to maintain consistency between other supported frameworks like Django and Flask.

Reply:
`plugin` is just a Bottle convention, although to be consistent with rest of middlewares it could be renamed

Reply:
regarding dockstring: https://www.python.org/dev/peps/pep-0257/#one-line-docstrings
or would you insist on multi-line style?
In my opinion using Black (https://github.com/psf/black) for a project would eliminate style/formatting questions and save time of a reviewer.

Reply:
For this PR we should maintain consistency. Thanks for the suggestion using Black. I will look into it.